helix-term/src/application.rs

```diff
@@ -1202,7 +1202,11 @@ impl Application {
             .backend_mut()
             .show_cursor(CursorKind::Block)
             .ok();
-        self.terminal.restore(terminal_config)
+
+        use std::io::Write;
```

Comment:
I'd suggest to move this import to the top of the file and not inside the function.